### DISCUSSION TITLE
I want to apply fomantic-ui to index.pug that is made of npm for node.js

### DISCUSSION THREAD

**Discussion by kimnakyeong** [fomantic/Fomantic-UI#792] [General] [answered]
# Help Wanted

## Problem
Describe your problem in great detail

    script(src='https://cdn.jsdelivr.net/npm/jquery@3.3.1/dist/jquery.min.js')
    link(rel='stylesheet', type='text/css', href='../semantic/dist/semantic.css')
    script(src='../semantic/dist/semantic.js')

this is for index.pug and 

semantic folder is up of index.pug. so i add '..'

but it doesn't working and I googled it but I don't know what is wrong. 

please I need help.

**Comment by y0hami** [comment]
@kimnakyeong Is your web server serving the semantic files? Your path in the `link` and `script` tag will have to be relative for the server not your project structure.

**Comment by nektro** [comment]
Try this instead:
```pug
link(rel="stylesheet", type="text/css", href="https://cdnjs.cloudflare.com/ajax/libs/fomantic-ui/2.7.5/semantic.min.css")
script(src="https://cdnjs.cloudflare.com/ajax/libs/fomantic-ui/2.7.5/semantic.min.js")
```

**Comment by y0hami** [comment]
@nektro That may not work in this case since they might be installing the package from NPM so they can create a theme and you can't create themes when using the CDN.

**Comment by lubber-de** [comment]
Is this still an issue or can we close this?

**Comment by lubber-de** [comment, marked as answer]
@kimnakyeong
If your folder structure looks like this:
```
semantic
     | dist
        |- semantic.js
        |- semantic.css
index.pug
```
Then do _not_ use the parent folder ".." but

```pug
  script(src='https://cdn.jsdelivr.net/npm/jquery@3.3.1/dist/jquery.min.js')
  link(rel='stylesheet', type='text/css', href='semantic/dist/semantic.css')
  script(src='semantic/dist/semantic.js')
```
instead.

Using ".." would be correct if your folder structure looks like this

```
semantic
     | dist
        |- semantic.js
        |- semantic.css
mysite
     |- index.pug
```

However, if you use the dist versions of Fomantic and dont want to create own themes (as @hammy2899 mentioned above) and already use a cdn for jquery you may use cdns for fomantic as well (then your folder structure does not matter)

```pug
  script(src='https://cdn.jsdelivr.net/npm/jquery@3.3.1/dist/jquery.min.js')
  link(rel='stylesheet', type='text/css', href='https://cdn.jsdelivr.net/npm/fomantic-ui@2.7.6/dist/semantic.min.css')
  script(src='https://cdn.jsdelivr.net/npm/fomantic-ui@2.7.6/dist/semantic.minjs')
```

**Comment by lubber-de** [comment]
Closing because answers are given so far and no more response from the author since 6 weeks.
Will reopen if this is still an issue and the author provides more info
